package.json

```diff
@@ -19,43 +19,42 @@
     "ionic_webpack": "webpack.config.js"
   },
   "dependencies": {
-    "@angular/common": "2.4.8",
-    "@angular/compiler": "2.4.8",
-    "@angular/compiler-cli": "2.4.8",
-    "@angular/core": "2.4.8",
-    "@angular/forms": "2.4.8",
-    "@angular/http": "2.4.8",
-    "@angular/platform-browser": "2.4.8",
-    "@angular/platform-browser-dynamic": "2.4.8",
-    "@angular/platform-server": "2.4.8",
-    "@ionic-native/core": "^3.6.1",
+    "@angular/common": "4.1.3",
+    "@angular/compiler": "4.1.3",
+    "@angular/compiler-cli": "4.1.3",
+    "@angular/core": "4.1.3",
+    "@angular/forms": "4.1.3",
+    "@angular/http": "4.1.3",
+    "@angular/platform-browser": "4.1.3",
+    "@angular/platform-browser-dynamic": "4.1.3",
+    "@ionic-native/core": "3.12.1",
+    "@ionic-native/splash-screen": "3.12.1",
+    "@ionic-native/status-bar": "3.12.1",
+    "@ionic/storage": "2.0.1",
+    "ionic-angular": "3.6.0",
+    "ionicons": "3.0.0",
+    "rxjs": "5.4.0",
+    "sw-toolbox": "3.6.0",
+    "zone.js": "0.8.12",
     "@ionic-native/geolocation": "^3.6.1",
     "@ionic-native/keyboard": "^3.6.1",
-    "@ionic-native/splash-screen": "^3.6.1",
-    "@ionic-native/status-bar": "^3.6.1",
-    "@ionic/storage": "^2.0.0",
     "@types/google.analytics": "0.0.33",
     "haversine": "^1.0.2",
     "intl": "^1.2.5",
-    "ionic-angular": "2.2.0",
-    "ionicons": "3.0.0",
     "localforage": "1.4.0",
     "lodash": "^4.17.4",
     "moment": "^2.17.1",
-    "ng2-pagination": "^2.0.1",
-    "rxjs": "5.0.1",
-    "sw-toolbox": "3.4.0",
-    "zone.js": "0.7.2"
+    "ng2-pagination": "^2.0.1"
   },
   "devDependencies": {
-    "@ionic/app-scripts": "^1.3.7",
-    "@ionic/cli-build-ionic-angular": "^0.0.3",
-    "@ionic/cli-plugin-cordova": "^1.4.0",
-    "@ionic/cli-plugin-ionic-angular": "^1.3.1",
+    "@ionic/app-scripts": "2.1.3",
+    "@ionic/cli-plugin-cordova": "1.5.0",
+    "@ionic/cli-plugin-ionic-angular": "1.4.0",
     "@types/jasmine": "^2.5.41",
     "@types/node": "^7.0.8",
     "angular2-template-loader": "^0.6.2",
     "html-loader": "^0.4.5",
+    "ionic": "3.6.0",
```

Comment:
There was debate about this before (#378), it looks like Ionic doesn't `preferGlobal` anymore. README should be updated, and any chance we want to revisit the issue in general?